packages/browser/src/bugsnag.ts

```diff
@@ -116,7 +116,9 @@ const notifier: BrowserClient = {
 
 type Method = keyof typeof Client.prototype
 
-map(['resetEventCount'].concat(keys(Client.prototype)) as Method[], (m) => {
+const clientMethods = Object.getOwnPropertyNames(Client.prototype).concat(['resetEventCount']) as Method[]
```

Comment:
Well something has changed meaning that the class properties are now non-enumerable so `keys(Client.prototype)` (or `Object.keys`) no longer works.